README.md

```diff
@@ -1,4 +1,4 @@
-![continuous integration](https://github.com/tokio-rs/prost/workflows/continuous%20integration/badge.svg)
+[![continuous integration](https://github.com/tokio-rs/prost/actions/workflows/ci.yml/badge.svg?branch=master)](https://github.com/tokio-rs/prost/actions/workflows/ci.yml)
```

Comment:
The badge is for branch master. Should the link also be for branch master?

Reply:
The search query results in only showing CI results that the badge is referring to, so that makes more sense to me then showing CI results of random PRs. I leave it up to you.

Reply:
i guess it does make sense enough to show just master branch. and it's not like fragility based on whatever github decides to break matters much since that's kinda upstream of all the other parts of it anyway